### PR TITLE
Fixes #14150: Add some known issues in the known issue of the DSC doc

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/14150

### CODE REVIEW

File: 85_Plugins/10_DSC_plugin.txt [hunk 1]
```diff
@@ -189,14 +189,20 @@ Then reboot the system.
 
 ==== Known issues
 
-On the first run of the Rudder DSC agent CLI in a Powershell terminal, you may have the following error message:
+* On Windows, Rudder can only manage user password in clear text.
 
+* Mustache templating on Windows is very limited and does not support array iteration, there is an open ticket in the private associated Redmine: https://issues.rudder.io/issues/11380.
```
Comment (amousset) [inline 85_Plugins/10_DSC_plugin.txt]: We should not put a link to a private issue in public do

File: 85_Plugins/10_DSC_plugin.txt [hunk 2]
```diff
@@ -189,14 +189,20 @@ Then reboot the system.
...
+
+* Folders can not be shared via the "Rudder shared folder" with a Windows agent, only files can be shared.
```
Comment (amousset) [inline 85_Plugins/10_DSC_plugin.txt]: The *Technique File download (Rudder server)* technique

**Comment by Fdall** [comment]
Commit modified

File: 85_Plugins/10_DSC_plugin.txt [hunk 3]
```diff
@@ -191,9 +191,9 @@ Then reboot the system.
 
 * On Windows, Rudder can only manage user password in clear text.
 
-* Mustache templating on Windows is very limited and does not support array iteration, there is an open ticket in the private associated Redmine: https://issues.rudder.io/issues/11380.
+* Mustache templating on Windows is very limited and does not support array iteration, there is an open ticket in the private associated Redmine.
```
Comment (amousset) [inline 85_Plugins/10_DSC_plugin.txt]: no need to mention the private issue at all

Comment (amousset) [reply 85_Plugins/10_DSC_plugin.txt]: @Fdall ping

**Comment by Fdall** [comment]
Commit modified

**Pull request merged**